documentation/Functions.md

```diff
@@ -350,20 +350,32 @@ If the tutorial should be tested on the console environment, you have to specify
 ***
 
 ### displayContent <a name="displayContent"></a>
+This function is only used when you want to display content such as text, image or any file content in your tutorial.
+
 #### parameter
-1. The title of the step. 
-2. An array of json objects with files, content, or images to be rendered within the Katacoda step. The use for this function is to display an image and some descriptive text. No Katacoda syntax is allowed in the files or the content!
-3. (Optional) Path to the current directory where the user is located (relative to the workspace directory). Only needed if the directory is changed within this step.
+you need two parameters for this function.
+
+Example: displayContent(1st parameter, 2nd parameter) 
+
+1st parameter i.e. The title of the step. 
```

Comment:
I would prefer a list syntax or bullet points for the parameters as it was before. So it stands out better from the text